src/components/common/Extensions/UserHub/partials/TransactionsTab/partials/GroupedTransactionStatus.tsx

```diff
@@ -22,20 +22,21 @@ const GroupedTransactionStatus: FC<TransactionStatusProps> = ({ status }) => {
         'text-negative-400': failed,
       })}
     >
-      {pending && (
-        <SpinnerGap
-          className="ml-2.5 h-[0.8125rem] w-[0.8125rem] animate-spin text-blue-400"
-          size={14}
-        />
-      )}
       <PillsBase
         className={clsx({
           'bg-success-100 text-success-400': succeeded,
           'bg-negative-100 text-negative-400': failed,
           'bg-gray-100 text-gray-500': !succeeded && !failed,
+          'relative flex items-center': true,
```

Comment:
Hey @codeseker if you want to pass in a constant set of styles to `clsx`, you can pass those in as the 1st argument:

```js
className={clsx('relative flex items-center', {
  'bg-success-100 text-success-400': succeeded,
  'bg-negative-100 text-negative-400': failed,
  'bg-gray-100 text-gray-500': !succeeded && !failed,
})}
```

Reply:
I've just tested your changes and the UI as a bit broken now.

![Screenshot 2024-06-17 at 07 54 18](https://github.com/JoinColony/colonyCDapp/assets/50642296/c3c736ae-df82-4f61-8386-669d9927691a)

Can you please double check the styles and if possible, give my suggestions in the other comment thread a go? Can you also please post a screenshot of your changes for your next set of changes? So we can see the effect of your changes at a glance. Thank you